README.md

```diff
@@ -45,6 +45,67 @@ you can enable/disable for the command-line app, use the `--help` option:
 addons-linter --help
 ```
 
+### Import Linter API into another NodeJS application
+
+```js
+import {createInstance as createLinter} from 'addons-linter';
+
+async lint(sourceDir) {
```

Comment:
Using the `async` syntax (which is quite new and a lot of developers won't know) isn't very friendly, especially to novice developers. I don't know the `async` syntax well enough to understand this example as much as Promises-based one.

I think async devs will know how to translate it from Promises, but the inverse isn't true. We should provide Promise-based examples (either first or only).